### ISSUE TITLE
mock.call_args raise TypeError

### ISSUE THREAD

**Issue by RPing** [syrusakbary/snapshottest#94]
0.5.0 works, but 0.5.1 will raise TypeError
```python
def test_call_args(snapshot):
    mock = Mock(return_value=None)
    mock(3, 4)

    snapshot.assert_match(mock.call_args)
```
```python
================================================================== FAILURES ===================================================================
_______________________________________________________________ test_call_args ________________________________________________________________

snapshot = <snapshottest.pytest.PyTestSnapshotTest object at 0x108534160>, mocker = <pytest_mock.MockFixture object at 0x108534278>

    def test_call_args(snapshot, mocker):
        mock = Mock(return_value=None)
        mock(3, 4)

>       snapshot.assert_match(mock.call_args)

test_app/src/tests/genv/gutils/tests/test_retry.py:80:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../snapshottest/snapshottest/module.py:246: in assert_match
    self.assert_value_matches_snapshot(value, prev_snapshot)
../snapshottest/snapshottest/module.py:229: in assert_value_matches_snapshot
    formatter.assert_value_matches_snapshot(self, test_value, snapshot_value, Formatter())
../snapshottest/snapshottest/formatters.py:19: in assert_value_matches_snapshot
    test.assert_equals(formatter.normalize(test_value), snapshot_value)
../snapshottest/snapshottest/formatter.py:24: in normalize
    return formatter.normalize(value, self)
../snapshottest/snapshottest/formatters.py:43: in normalize
    return value.__class__(formatter.normalize(item) for item in iterator)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

cls = <class 'unittest.mock._Call'>, value = <generator object CollectionFormatter.normalize.<locals>.<genexpr> at 0x108509eb8>, name = ''
parent = None, two = False, from_kall = True

    def __new__(cls, value=(), name=None, parent=None, two=False,
                from_kall=True):
        name = ''
        args = ()
        kwargs = {}
>       _len = len(value)
E       TypeError: object of type 'generator' has no len()
```

**Comment by 2m** [comment]
Yup. Just got this one as well. Did you find a workaround?

**Comment by mbelang** [comment]
`pytest --snapshot-update` works all the time but doesn't save the update. I reverted to version `0.5.0`.

**Comment by ignacio-dl** [comment]
I changed the tests to check for the specific arguments

```python
snapshot.assert_match(logger.info.call_args[0][0])
```

**Comment by paulmelnikow** [comment]
Hi, could you check if this is fixed in the latest master?

**Comment by paulmelnikow** [comment]
I think it may have been fixed by #116.

**Comment by paulmelnikow** [comment]
Please feel free to reopen or ping the issue if #116 did not resolve this.
